tests/specs/relay-general-func.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { Page } from '@playwright/test';
+import test, { expect }  from '../fixtures/basePages'
+import { defaultScreenshotOpts } from '../utils/helpers';
+
+// using logged in state outside of describe block will cover state for all tests in file
+test.use({ storageState: 'state.json' })
+test.describe('Free - General Functionalities, Desktop', () => {
+
+    test.beforeEach(async ({ dashboardPage, page }) => {
+      await dashboardPage.open()
+      await checkForSignInButton(page)
+      await dashboardPage.deleteMask()   
+    });    
+  
+    test('Verify that the Header is displayed correctly for a Free user that is logged in, C1812639', async ({ dashboardPage }) => {
+      await expect(dashboardPage.header).toHaveScreenshot(
+        'dashboardHeader.png',
+        defaultScreenshotOpts
+      );
+    })
+  
+    test.fixme('Verify that the "Upgrade" button redirects correctly,  C1812640, 1808503', async ({ dashboardPage, page }) => {
```

Comment:
Is this one still broken? Anything I can help with? It seems to work for me.

```suggestion
    test('Verify that the "Upgrade" button redirects correctly,  C1812640, 1808503', async ({ dashboardPage, page }) => {
```

Reply:
nope fixed